test/cli/remote-dbt-init/ok-interactive/interaction.txt

```diff
@@ -1,14 +1,14 @@
-< Please enter Keboola Storage API host, eg. "connection.keboola.com".
-> %%TEST_KBC_STORAGE_API_HOST%%
-
-< Please enter Keboola Storage API token. The value will be hidden.
-> %%TEST_KBC_STORAGE_API_TOKEN%%
-
 < Please enter target name.
 > target1
 
 < Enter a name for a workspace to create:
 > dbt_source
 
+< Please enter Keboola Storage API host, eg. "connection.keboola.com".
+> %%TEST_KBC_STORAGE_API_HOST%%
+
+< Please enter Keboola Storage API token. The value will be hidden.
+> %%TEST_KBC_STORAGE_API_TOKEN%%
+
```

Comment:
Upravena logika, najprv sa validuju/ziskavaju argumenty pre operaciu, az tak sa vytvara spojenie zo Storage API.

Reply:
V tomto commande to bolo naopak.